doc/source/user/basics.rec.rst

```diff
@@ -576,12 +576,12 @@ Record Arrays
 
 As an optional convenience numpy provides an ndarray subclass,
 :class:`numpy.recarray`, and associated helper functions in the
-:mod:`numpy.rec` submodule, that allows access to fields of structured arrays
-by attribute instead of only by index. Record arrays also use a special
-datatype, :class:`numpy.record`, that allows field access by attribute on the
-structured scalars obtained from the array.
+:mod:`numpy.lib.recfunctions` submodule, that allows access to fields of
```

Comment:
```suggestion
:mod:`numpy.lib.recfunctions` submodule (aliased as ``numpy.rec``), that allows access to fields of
```

Reply:
(since otherwise the reference to the `numpy.rec` spelling is lost by this PR)

Reply:
done